lib/APIPlugin.js

```diff
@@ -30,8 +30,18 @@ const REPLACEMENT_TYPES = {
 	__webpack_nonce__: "string"
 };
 /* eslint-enable camelcase */
+/** @typedef {import("./Parser")} Parser */
```

Comment:
Could you move this right after the `require` calls?

Reply:
Of course np. I followed the pattern of Sean in `chunk.js`, he moved the typings after "all" `const` decalarations..
Will update it

Reply:
done